app/presenters/vendor_api/notes_presenter.rb

```diff
@@ -0,0 +1,25 @@
+module VendorAPI
+  module NotesPresenter
```

Comment:
I think this should be named differently and could possibly by in the concerns folder as it is using the application presenter and adding to it rather than introducing a new presenter?

Reply:
see https://github.com/DFE-Digital/apply-for-teacher-training/pull/6416/files#diff-95ff1696f96b07dc50f82e57b3e454dac872150a1f673d70b33c7f63d7616fc7R1 and https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/docs/versioning_vendor_api.md